### PR TITLE
Enable expeditor for builds and versioning

### PR DESCRIPTION
**Pull request by scotthain**
Signed-off-by: Scott Hain <shain@chef.io>

### Description

Briefly describe the new feature or fix here

--------------------------------------------------

#### Maintainers

Please ensure that you check for:

- [] If this change impacts git cache validity, it bumps the git cache
  serial number
- [] If this change impacts compatibility with omnibus-software, the
  corresponding change is reviewed and there is a release plan
- [] If this change impacts compatibility with the omnibus cookbook, the
  corresponding change is reviewed and there is a release plan

### CODE REVIEW

**Comment by scotthain** [comment]
@tduffield

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  notify_channel: omnibus-rfr
+
+github:
+  maintainer_group: chef/omnibus-maintainers
+  version_tag_format: "v{{version}}"
+
+merge_actions:
+  - built_in:bump_version:
+      ignore_labels:
+        - "Version: Skip Bump"
```
Comment (schisamo) [inline .expeditor/config.yml]: Idiomatically we usually add a `Expeditor: Skip All` under each merge action. I wonder if we should consider adding this as built-in behavior @tduffield ?

File: .expeditor/config.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+slack:
+  notify_channel: omnibus-rfr
+
+github:
+  maintainer_group: chef/omnibus-maintainers
+  version_tag_format: "v{{version}}"
```
Comment (schisamo) [inline .expeditor/config.yml]: We should add the following under here:

```YAML
minor_bump_labels:
    - "Version: Bump Minor"
```

**Comment by scotthain** [comment]
@tduffield @schisamo I think I've incorporated your feedback - thanks!

File: .expeditor/config.yml (hunk 1 above)
Comment (tduffield) [reply .expeditor/config.yml]: @schisamo I go back and forth on that. My initial design intention was to be very explicit in this YAML about what labels do what.

File: .expeditor/config.yml [hunk 3]
```diff
@@ -4,13 +4,24 @@ slack:
...
         - built_in:bump_version
-  - built_in:build_gem
+  - built_in:update_changelog:
+      ignore_labels:
+        - "Changelog: Skip Update"
+        - "Expeditor: Skip All"
+  - built_in:build_gem:
+      only_if:
+        - built_in:bump_version
+      ignore_labels:
```
Comment (jaymalasinha) [inline .expeditor/config.yml]: The ignore_labels here would not take effect due to the ```only_if``` right.  Do we still need it here ?

Comment (scotthain) [reply .expeditor/config.yml]: You're right, we don't.

**Pull request merged**